src/_select.scss

```diff
@@ -0,0 +1,111 @@
+// Select
+// ===
+
+
+// Configurable Variables
+// ---
+//
+// 1. The same as default side paddings (but they're set in 1 )
+
+$select__height: if(variable-exists(forms__element-height), $forms__element-height, 50px) !default;
+$select__v-space: if(variable-exists(v-space), $v-space, 0.5rem) !default;
+$select__h-space: if(variable-exists(h-space), $h-space, 1rem) !default;
+$select__padding: if(variable-exists(forms__element-padding), $forms__element-padding, 10px 15px) !default;
+
+$select__border-color: if(variable-exists(border-color), $border-color, hsl(0, 0%, 10%)) !default;
+$select__border: if(variable-exists(border), $border, 1px solid $select__border-color) !default;
+$select__border-radius: if(variable-exists(forms__border-radius), $forms__border-radius, 0) !default;
+
+$select__background-color: if(variable-exists(forms__background-color), $forms__background-color, #fff) !default;
+$select__triangle-size: 7px !default;
+$select__triangle-position: 15px !default; // 1
+
+$select__font-family: if(variable-exists(forms__font-family), $forms__font-family, inherit) !default;
+$select__font-size: if(variable-exists(forms__font-size), $forms__font-size, inherit) !default;
+$select__line-height: 20px;
```

Comment:
So here’s where we have some serious issues. For such a simple component, this is *way* overkill on config variables. Since the new Stencil will have a lot more components, imagine what will happen on a full project. On EBS we have 63 components in the build. If 50% are from Stencil 2.0 and have this many config vars, that’s ~410 variables to grok just to set up your components to match the project.

I am starting to doubt whether config vars are going to scale. I suspect the best approach will be to build the components with as few style declarations as possible for them to work, and write a separate theme on top for use in prototyping or banging out a super-constrained generic-looking design.